### DISCUSSION TITLE
pH7Builder v17.1.2 is available 🥳

### DISCUSSION THREAD

**Discussion by pH-7** [pH7Software/pH7-Social-Dating-CMS#980] [Release Announcements]
**pH7Builder v17.1.2** is now available. 

This version minor version addresses the following changes:
*✔️ Removing a leftover file when the "game" module got removed - https://github.com/pH7Software/pH7-Social-Dating-CMS/commit/bcc92ec1bb8bc36cbc0d02006b562dd0a98d086d
* ✔️ Not allow to select "No countries" for the available countries on the registration form and search forms - https://github.com/pH7Software/pH7-Social-Dating-CMS/commit/4b846a25ff21fa0ee923f471e6d980206ee275a0 - This will help to prevent issues if you mistakenly save No Country in the settings as this option isn't possible.
* ✔️ Fix incorrect return type declaration - https://github.com/pH7Software/pH7-Social-Dating-CMS/commit/83647a61480497255044a3b168849e67d70188ee
* ✔️ Rename the default PHP error log file to plural - https://github.com/pH7Software/pH7-Social-Dating-CMS/commit/e518dd638ecccccba28d5ce00bc78e5db31c2b11

Enjoy your **brand new social dating builder** 🎉

<hr /><em>This discussion was created from the release <a href='https://github.com/pH7Software/pH7-Social-Dating-CMS/releases/tag/17.1.2'>pH7Builder v17.1.2 is available 🥳</a>.</em>

**Comment by norspeed22** [comment]
I am trying to install the lastest version17.1.2, however, despite having provisioned the server for PDO, and exif, i am still getting the following errors: You have 3 error(s): and unable to install, any ideas as to why ?

❌ 1) Please install "PDO" PHP extension.
❌ 2) Please install "exif" PHP extension.
❌ 3) Please install "exif" PHP extension.
